scrapers/nation.py

```diff
@@ -26,26 +28,33 @@ def scrape_page(self):
         if result_html:
             data = []
             items = result_html.find_all(
-                "div", class_="story-teaser medium-teaser")
+                "div", class_="story-teaser top-teaser")
             for item in items:
                 img_src = item.find("img").get("src")
                 if img_src:
                     img_url = base_urls["nation"] + img_src
-                img_url = "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png"
+                else:
+                    img_url = "https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/placeholder.png"
                 link = base_urls["nation"] + item.find("a").get("href")
                 text = item.find("img").get("alt")
                 data.append({
                     'link': link,
                     'img': img_url,
                     'title': text
                 })
-            print (json.dumps(data, indent=2))
-            self.base.s3.put_object(
-                Bucket='taxclock.codeforkenya.org',
-                ACL='public-read',
-                Key='data/standard-news.json',
-                Body=json.dumps(data)
-            )
-            return result_html
+            print (json.dumps(data[:2], indent=2))
+            try:
+                self.base.s3.put_object(
+                    Bucket='taxclock.codeforkenya.org',
+                    ACL='public-read',
+                    Key='data/standard-news.json',
```

Comment:
This seems to write to the same file and so do the others that are not Standard scrapers.

Reply:
@DavidLemayian 
I'll use different keys for the other scrapers.